Dockerfile

```
@@ -4,19 +4,11 @@ ENV APACHE_DOCUMENT_ROOT=/app
 
 RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini" \
 	&& apt-get --yes update && apt-get --yes install libxml2-dev zlib1g-dev libmemcached-dev \
```

```suggestion
	&& apt-get update -qy \
	&& apt-get install --no-install-recommends -qy \
		libmemcached-dev \
		zlib1g-dev \
```

``--no-install-recommends`` is recommended to only install needed packages.
``-qy`` is shorter than --yes and also hides some useless output.
``libxml2-dev`` is not needed when simplexml is not compiled.
Putting the packages in multi lines makes merge diffs easier to handle and less likely to conflict in the future and it is more clear which exact packages get installed.

---

IMHO in scripts you should always use the long-hand variants of options. The shorthands are there to save you some typing in exchange for readability. But scripts should be readable.

---

I think this is generally true for most flags but -q -y and -v are so general that you should know what they do. For other stuff like -n, -P or -ti you probably should use the long form.
but I must admit that this is probably irrelevant in this case.

---

Kept the `--long-flags`. Added `--no-install-recommends`. Not sure about `-q`, is build verbosity really an issue?

---

🤷‍♂️
TBH it is not that important.